release-notes/v0.6.0.md

```diff
@@ -0,0 +1,153 @@
+# What's new in Visual Studio Code Java?
+
+*February 2019*
+
+Welcome to the February 2019 release of Java Extension Pack. This document aims to recap the features, bug fixes around Java language support in between releases. Here are some of the key highlights:
+
+- **[IntelliCode](#intellicode)** for faster coding assisted by AI
+- **[Standalone Java Files](#standalone-java-files-supported)** better supported
+- **[Performance](#performance)** improved
+- **[Dependency Auto-Completion](#dependency-auto-completion-for-pom.xml-files)** for editing `pom.xml` files
+
+The release notes are arranged in the following sections releated to VS Code Java focus areas. Here are some further updates:
+
+- **[Code Editing](#code-editing)**
+- **[Source Code Management](#source-code-management)**
+- **[Performance](#performance)**
+- **[Maven](#maven)**
+- **[Debugging](#debugging)**
+- **[Testing](#testing)**
+
+As always, please feel free to [tweet us your feedback](https://twitter.com/intent/tweet?via=code&hashtags=Java%2CHappyCoding) or [open issues](https://github.com/Microsoft/vscode-java-pack/issues).
+
+## Code Editing
+
+## IntelliCode
+
+IntelliCode saves you time by putting what you’re most likely to use at the top of your completion list. IntelliCode recommendations are based on thousands of open source projects on GitHub each with over 100 stars. When combined with the context of your code, the completion list is tailored to promote common practices.
+
+IntelliCode works well with popular Java libraries and frameworks like Java SE and Spring. It will help you whether you are doing monolithic web apps or modern microservices.
+
+In the sample below, some of the completion lists have items marked with ★. These items are the recommendations provided by IntelliCode according to the current code context.
+
+![IntelliCode](v0.5.0/intellicode.gif)
+
+**Note:** TypeScript/JavaScript, Python, C++, C# and XAML are also supported.
+
+### Generate Getters & Setters
+
+Now you can generate getters and setters for new member variables.
+
+![Generate Getters & Setters](v0.6.0/gettersettergen.gif)
```

Comment:
The gif you provide is a quick fix. Actually we also want to mention the bulk generate via context menu "Source Action-> Generate Getters and Setters".

![image](https://user-images.githubusercontent.com/14052197/52990430-1b7ada00-3443-11e9-818f-8e99ac9ba3dd.png)
![image](https://user-images.githubusercontent.com/14052197/52990447-2f264080-3443-11e9-8e7f-26e89e7a0142.png)